cli/server/server.go

```diff
@@ -100,7 +100,7 @@ func NewCommands() []*cli.Command {
 			Subcommands: []*cli.Command{
 				{
 					Name:      "dump",
-					Usage:     "Dump blocks (starting with block #1) to the file",
```

Comment:
Why do you reference the issue in the commit header?

Reply:
Because it fits in there? Usually the problem is that your 60-80 symbols is not enough for it.